.github/CONTRIBUTOR_GUIDELINES.md

```diff
@@ -0,0 +1,110 @@
+
+To make the contribution process as seamless as possible, we ask for the following:
+
+* Go ahead and fork the project and make your changes.  We encourage pull requests to allow for review and discussion of code changes.
+* When you’re ready to create a pull request, be sure to:
+    * Have test cases for the new code. If you have questions about how to do this, please ask in your pull request.
+    * Add documentation if you are adding new features or changing functionality.  The docs site lives on [opencollarteam.github.io](https://github.com/OpenCollarTeam/opencollarteam.github.io).
+    * Squash your commits into a single commit. `git rebase -i`. It’s okay to force update your pull request with `git push -f`.
+    * Ensure that `lslint` succeeds. [GitHub Actions](https://github.com/OpenCollarTeam/OpenCollar/actions)  will fail the build if `lslint` fails.
+    * Follow the **Git Commit Message Guidelines** below.
+
+### Git Commit Message Guidelines
+
+This [blog article](http://chris.beams.io/posts/git-commit/) is a good resource for learning how to write good commit messages,
+the most important part being that each commit message should have a title/subject in imperative mood starting with a capital letter and no trailing period:
+*"Return error on wrong use of the Paginator"*, **NOT** *"returning some error."*
+
+Also, if your commit references one or more GitHub issues, always end your commit message body with *See #1234* or *Fixes #1234*.
+Replace *1234* with the GitHub issue ID. The last example will close the issue when the commit is merged into *master*.
+
+Sometimes it makes sense to prefix the commit message with the package name (or docs folder) all lowercased ending with a colon.
+That is fine, but the rest of the rules above apply.
+So it is "meta: Add new contributor guidelines", not "this adds new contributor guidelines", and "docs: Document emoji", not "doc: document emoji."
+
+Please use a short and descriptive branch name, e.g. **NOT** "patch-1". It's very common but creates a naming conflict each time when a submission is pulled for a review.
+
+Use the following sentence as a guide for writing the commit message:
+`If applied, this commit will [your text]`
+
+
+
+An example:
+
+```text
+meta: Compile LSLint on every workflow dispatch
+
+Remove static linking of lslint binaries, rather clone and build the tool automatically on execution.
+
+PR-URL: #502
+```
+
+### Linting and Testing OpenCollar with Your Changes
+
+OpenCollar uses [lslint](https://github.com/Makopo/lslint) to check the syntactic and semantic validity of the code. Additionally, you should test your changes in-world and make sure that no changes modify any other, unrelated component.
+
+Download [lslint from the releases page](https://github.com/Makopo/lslint/releases)
+
+To lint OpenCollar:
+
+```bash
+./run_lslint.sh /path/to/lslint/binary
+```
```

Comment:
Add a section: ### Reviewing Others' Pull Requests
This is an important way to help OpenCollar, since pull requests from contributors will not be merged without review.

1.  Check the github for outstanding pull requests.
2. Read the pull request
3. Review and either mark approval or suggest further changes.